README.md

```diff
@@ -172,6 +172,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Build Slate documentation](https://github.com/Decathlon/slate-builder-action)
 - [Read Properties](https://github.com/christian-draeger/read-properties) - Read values from `.properties` files.
 - [Write Properties](https://github.com/christian-draeger/write-properties) - Write values to `.properties` files.
+- [Autotag](https://github.com/butlerlogic/action-autotag) - Automatically generate a new tag when the manifest file (i.e. `package.json`) version changes.
```

Comment:
I feel this would be better categorized in the "GitHub Tools and Management" section because it fits under a utilization of GitHub resources and not necessary a third parties resources. Any thoughts surrounding that notion? 
Also, it may be more helpful for users if the link title was a little more specific. Something along the lines of "Autotag with Manifest Version Changes"?

Reply:
Yeah, agreed, that makes sense

Reply:
I've moved this, per the suggestion.